src/dialogsettings.ui

```diff
@@ -745,7 +745,7 @@
                </sizepolicy>
               </property>
               <property name="text">
-               <string>Thunderbird command:</string>
+               <string>Thunderbird command line:</string>
```

Comment:
It looks like you forgot to commit the changes in the language files.

Reply:
I didn't change language files, as I've no idea what to do with translations - I never use/build them.

Reply:
The language files are automatically changed when you build the project with cmake if you have the language tools installed.